src/prototype_room_my.js

```diff
@@ -242,7 +242,10 @@ Room.prototype.executeRoom = function() {
 
   if (spawns.length === 0) {
     this.reviveRoom();
-  } else if (this.energyCapacityAvailable < 1000) {
+  }
+  // Revive Room if energycapacity or storage don't met minimum
+  else if (this.energyCapacityAvailable < 1000 ||
+  this.storage.store.energy < config.nextRoom.minStorage) {
```

Comment:
I sometimes ran into the case that the room storage was empty - Usually messed up something, recovers after some time. This would trigger nextroomers where it is not necessary / helpful.

But maybe I shouldn't mess up stuff :-) And if it happens I should fix it and get reminded by seeing the nextroomers :-) (Or I change the config to 0).

So I'm fine with it, just want to mention it.